doc/whats_new.rst

```diff
@@ -43,6 +43,9 @@ Changelog
   returned for cases when there should be no valid matches, by `George Dang`_
   in :gh:`739`
 
+- Added kwargs options to `plot_spikes_hist` for adjusting the histogram plots of spiking activity,
+  by `Abdul Samad`_ in :gh:`732`.
```

Comment:
Oh, and your name here isn't consistent with your entry in the other PR so the link won't work.